packages/fxa-auth-server/scripts/email-config.js

```diff
@@ -7,7 +7,7 @@
 const ROOT_DIR = '..';
 const LIB_DIR = `${ROOT_DIR}/lib`;
 
-const config = require(`${ROOT_DIR}/config`).getProperties();
+const config = require(`${ROOT_DIR}/config`).default.getProperties();
```

Comment:
Random comment, do you ahve any thoughts about cleaning up these paths? It breaks the 'goto' in VSCODE for me.

Reply:
I think vscode only understands string literal for something like this?  if so then we have only one option.